.github/workflows/build-container.yml

```
@@ -199,6 +199,19 @@ jobs:
       - name: Display logs
         run: make logs OPTS=''
 
+      - name: Login to quay.io (on release/tag create)
+        if: startsWith(github.ref, 'refs/tags/v')
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: '${{ secrets.QUAY_IO_USERNAME }}'
+          password: '${{ secrets.QUAY_IO_PASSWORD }}'
```

Let's make it registry neutral then (note the `vars`, as `secrets` is only for passwords and sensitive info):
```suggestion
          username: '${{ vars.CONTAINER_REGISTRY_USERNAME }}'
          password: '${{ secrets.CONTAINER_REGISTRY_PASSWORD }}'
```
I also created these vars/secrets.

---

Done